rust/private/common.bzl

```diff
@@ -25,6 +25,10 @@ In the Bazel lingo, `rust_common` gives the access to the Rust Sandwich API.
 
 load(":providers.bzl", "CrateInfo", "DepInfo", "StdLibInfo")
 
+# Note: Code in `.github/workflows/crate_universe.yaml` looks for this line, if you remove it or change its format, you will also need to update that code.
+DEFAULT_RUST_VERSION = "1.54.0"
```

Comment:
Please add a comment saying that this is a default that can be overriden by the toolchain (and in the future I can see we will have a build setting that will override even the toolchain). Same for the edition.

Reply:
Updated. Hope it's what you were thinking 😄